src/shared/components/catalog/catalog.scss

```diff
@@ -5,8 +5,8 @@ $catalog-capability-level-icon-top: 4px;
 $catalog-capability-level-inactive-color: var(--pf-global--Color--400);
 $catalog-item-icon-size-lg: 40px;
 $catalog-item-icon-size-sm: 24px;
-$co-modal-ignore-warning-icon-width: 30px;
-$catalog-tile-width: $co-m-catalog-tile-width;
+$hacDev-modal-ignore-warning-icon-width: 30px;
```

Comment:
I think we can break this file into multiple scss files co-located with the components using individual pieces.

Reply:
Break this Catalog.scss into multiple scss files and co-locate with components